lib/datadog/di/probe_notification_builder.rb

```diff
@@ -46,7 +46,9 @@ def build_executed(context)
         build_snapshot(context)
       end
 
-      NANOSECONDS = 10**9
+      # Steep: `Integer ** Integer` can also result in Rational (e.g. 10**-2)
+      # which is why it gives an error if NANOSECONDS is set to 10**9.
```

Comment:
Minor: I suggest removing this comment. That's because, if you came across the new code, you'd never ask "Wait, why is this not 10**9 instead of 1_000_000_000, I must know the reason!!" ;)

I do understand the interest for this kind of comment, but this is a comment for the PR! Or at least as a commit message -- because it's a comment about the change from one to another. But going forward, we'll never need to care about this again.

```suggestion
```